scripts/dashboard/install-e2e-test-deps.sh

```diff
@@ -7,6 +7,29 @@ if [[ ! $(arch) =~ (i386|x86_64|amd64) ]]; then
 fi
 
 if grep -q  debian /etc/*-release; then
+    NODEJS_MAJOR_VERSION=14
+    DISTRO="$(lsb_release -cs)"
+    if [[ ! $(command -v node) || $(node --version | grep -oE "v([0-9])+" | cut -c 2-) < ${NODEJS_MAJOR_VERSION} ]]; then
+        sudo apt remove nodejs -y
+        sudo apt remove nodejs-doc -y
+        sudo dpkg --remove --force-remove-reinstreq libnode-dev
+		sudo dpkg --remove --force-remove-reinstreq libnode72:amd64
+        sudo add-apt-repository -y -r ppa:chris-lea/node.js
```

Comment:
Why is this necessary?  Are the packages different than what's in the `https://deb.nodesource.com/${NODEJS_VERSION} ${DISTRO} main` repo?

Reply:
this was needed because at the time of the implementation, the ubuntu version we had required us to add the `chris-lea/node.js` repo too. And now since the ubuntu machines are updated we can remove it.

Reply:
and i think it came from here: https://github.com/ceph/ceph-build/pull/1932#discussion_r746623637

Reply:
btw, i removed it. do you think i need to add it back?

Reply:
No, I'd rather not use a ppa